### ISSUE TITLE
R197 - Status menu and mobile bar menu appearing on Android devices.

### ISSUE THREAD

**Issue by lumicreative** [Scirra/Construct-3-bugs#3810]
## Problem description

When checking hiding status bar during export, it hides the status bar. Now with the new update, status bar and mobile buttons (back, home, etc..) appear.

## Attach a .c3p

[hideStatusBar.c3p.zip](https://github.com/Scirra/Construct-3-bugs/files/4542416/hideStatusBar.c3p.zip)

## Steps to reproduce

1. Export as APK and ensure Hide Status Bar is checked
2. Install on Android mobile device
3. Open app

## Observed result

Status bar is still showing

## Expected result

Status bar and menu bar should be hidden.

## More details
- Tested on Android device (Note8 running latest OS)

**Affected browsers/platforms:** 

N/A

**First affected release: Worked in r189.2 but broke in r197

## System details

N/A

**Comment by fahidhv** [comment]
I'm also facing this issue, it's from previous beta release and still not fixed!

**Comment by AshleyScirra** [comment]
This was unintentionally broken in r195 when we adjusted the combination of Cordova plugins applied when changing how viewport fit 'Cover' is used. We had 2 weeks of this being tested on the beta channel during which time there were hundreds of Android exports, but it looks like despite that it slipped through the net.

I've got a fix for this and we'll probably have a patch update some time this week. Leaving this issue open and marked fixed since I've seen this being mentioned a few times already.
